apps/docs/content/components/Image.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Image
+status: Experimental
+source: https://github.com/primer/brand/blob/main/packages/react/src/Image/Image.tsx
+storybook: '/brand/storybook/?path=/docs/components-image--playground'
+description: Use Image to create both normal and responsive image elements.
+---
+
+import {Label} from '@primer/react'
+
+```js
+import {Image} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+```jsx live
+<Image src="https://source.unsplash.com/random" alt="Random Image" />
+```
+
+### Picture
+
+The `as` prop can be used to set the container of the image to be use `picture`.
+
+```jsx live
+<Image
```

Comment:
If I wanted to generate this markup:

```html
<picture>
  <source srcset="logo-768.png, logo-768-1.5x.png 1.5x" />
  <img src="logo-320.png" alt="logo" />
</picture>
```

How would I do that with as="picture"? I can't see a way to set `sources`.

Reply:
Currently it sets on the image tag but I could include the children prop. Do you think that would be beneficial? I wasn't sure if that was really more performant.

Reply:
We'd deffo need a way to produce the `<source>` elements when setting `as="picture"`. I don't think we need to do it via JSX children, but it should render markup like the example above. Could you show me me how you saw it working currently without doing that?

Reply:
Basically you would apply the property to the image. What I can do it parse the property from the img component and then take each value to create a source component. Would that work for you?